src/ray/util/test_util.h

```diff
@@ -58,6 +60,34 @@ std::shared_ptr<RayObject> GenerateRandomObject() {
   return std::shared_ptr<RayObject>(new RayObject(GenerateRandomBuffer(), nullptr));
 }
 
+/// Path to redis server executable binary.
+std::string REDIS_SERVER_EXEC_PATH;
+/// Path to redis client executable binary.
+std::string REDIS_CLIENT_EXEC_PATH;
+/// Path to redis module library.
+std::string REDIS_MODULE_LIBRARY_PATH;
```

Comment:
Can we put these above variables as public fields of `ManageRedisServiceForTest` instead?

Right now they are referenced as `ray::REDIS_SERVER_EXEC_PATH,` it's not clear that they are actually only for tests.

Reply:
Sounds good.

Reply:
I just recall why i didn't make those `vars` as `member` of `ManageRedisServiceForTest`. Those `vars` will used in `gtest` static function `SetUpTestCase` to setup redis service. So those `vars` only can be `static variable` or `global variable`.  Obviously `static variable` not ok.